### PR TITLE
Start self-help troubleshooting

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

**Comment by mrkschan** [comment]
Will add links to other docs later.

File: src/development/troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+## HTTP 502 response
+
+If you're receiving HTTP 502 responses from your site,
+that means your site is running out of workers.
+Here are the typical causes:
+
+* The amount of traffic coming to your site exceeds
+  the processing power of your application.
```
Comment (Crell) [inline src/development/troubleshoot.md]: No hard-wordwrap.

**Comment by mrkschan** [comment]
Updated.

**Comment by Crell** [comment]
Thanks, KS.  Looks good overall.  Two notes:

1) Please use soft-wraps, not hard-wraps, for all pages.

2) Can we remove anything from the FAQ page(s) now as a result of this page?  If so, we should.  FAQ pages need to go away.

**Comment by mrkschan** [comment]
Updated, will squash the commits once approved.

@Crell

**Comment by mrkschan** [comment]
softwrapping at paragraph now.

**Comment by mrkschan** [comment]
Used Squash and Merge option on this PR :p

**Pull request merged**